src/mame/machine/vsnes.cpp

```diff
@@ -255,10 +256,34 @@ MACHINE_START_MEMBER(vsnes_state,vsdual)
 	m_ppu1->space(AS_PROGRAM).install_read_bank(0x0000, 0x1fff, "bank2");
 	// read only!
 	m_ppu2->space(AS_PROGRAM).install_read_bank(0x0000, 0x1fff, "bank3");
-	membank("bank2")->configure_entries(0, m_vrom_size[0] / 0x2000, m_vrom[0], 0x2000);
-	membank("bank3")->configure_entries(0, m_vrom_size[1] / 0x2000, m_vrom[1], 0x2000);
-	membank("bank2")->set_entry(0);
-	membank("bank3")->set_entry(0);
+	m_bank_vrom[0] = membank("bank2");
+	m_bank_vrom[1] = membank("bank3");
```

Comment:
If you’re going to be keeping these in members, why not use object finders rather than searching for them here?

Reply:
That's what I attempted to do at first, but it seems like the preceding calls to `install_read_bank` cause the banks to only be created after memory bank finders would have all already been resolved. Maybe there's a different way to go about it though?

Reply:
Ah, it’s installing stuff in initialisation functions rather than using configured memory maps.  That would mean the memory banks don’t exist when object finders are resolved.

Since this system isn’t cartridge-based, it could be converted to use configured memory maps (we want to get away from installing handlers in initialisation functions when it isn’t actually necessary), but that would be a pile more work to do.  It’s probably a bit much unrelated work to worry about for this pull request.